proto/accessgraph/v1alpha/aws.proto

```diff
@@ -517,10 +517,12 @@ message AWSSAMLProviderV1 {
   repeated AWSTag tags = 4;
   // account_id is the ID of the AWS account that the provider belongs to.
   string account_id = 5;
-  // entity_id is the SAML entity ID for this provider
+  // entity_id is the SAML entity ID for this provider.
   string entity_id = 6;
-  // sso_urls is a list of single sign-on service URLs for this identity provider
+  // sso_urls is a list of single sign-on service URLs for this identity provider.
   repeated string sso_urls = 7;
+  // signing_certificates is a list of signing certificates for this identity provider.
```

Comment:
What are signing_certificates used for? From reading this PR alone, I cannot tell why we need them. 🤔

Reply:
On the TAG side, we'd like to match "identity provider" defined on the AWS (relying party) side with the actual identity provider.

I looked into what could be used as a unique identifier to match them up, and SAML Entity ID was not sufficient, because e.g. in Azure / Entra, under one Entra tenant (directory), you can have multiple enterprise applications (essentially identity providers) and they will have the same Entity ID, but different SAML certificate sets. So for now we're using the SAML certificates as this unique identifier.